### ISSUE TITLE
Строчный вывод

### ISSUE THREAD

**Issue by senddownrain** [Ivan-Shymanovich/tryruby#3]
0/8 - походу так выводить неправильно 
1/1 
4/4
- как ты считаешь, так выводить нормально?

**Comment by Ivan-Shymanovich** [comment]
Согласен, выводит некорректно. Пытался исправить это в методе #to_s, но ничего у меня не получилось. Это нужно обсудить, нужны подсказки, у меня идей нет уже.

**Comment by senddownrain** [comment]
думаю, просто можно делать проверку на результат операции mod
в руби походу это метод modulo
если остаток от деления равен 0 то выводить результат деления числитель на знаменатель.
если нет - то обычный вывод

**Comment by Ivan-Shymanovich** [comment]
Насчет этого еще нужно подумать, но уже лучше.

**Comment by Ivan-Shymanovich** [comment]
Нужны еще доработки, нет идей какую проверку делать.
